<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QLineEdit" name="secret_key_edit"/>
   </item>
   <item row="5" column="1">
    <widget class="QDialogButtonBox" name="buttons">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="name_lbl">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="secret_key_lbl">
     <property name="text">
      <string>Secret Key:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="access_key_lbl">
     <property name="text">
      <string>Access Key:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="access_key_edit"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="name_edit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="default_region_lbl">
     <property name="text">
      <string>Default Region:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="default_region_edit">
     <property name="text">
      <string>eu-central-1</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>name_edit</tabstop>
  <tabstop>access_key_edit</tabstop>
  <tabstop>secret_key_edit</tabstop>
  <tabstop>default_region_edit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
